### ISSUE TITLE
Test can run "raxml" - failed

### ISSUE THREAD

**Issue by ghbhc** [davidemms/OrthoFinder#538]
Hello,

When trying to run OrthoFinder as such:

`orthofinder -t 12 -a 12 -f ./proteomes -M msa -A mafft -T raxml`

I'm getting: 

"Test can run "raxml" - failed

ERROR: Cannot run user-configured tree method 'raxml'
Please check program is installed and that it is correctly configured in the orthofinder/config.json file"

I've confirmed that the config.json looks exactly like it's recommended to here: https://github.com/davidemms/OrthoFinder/issues/93, and that it exists in the location var directory as defined in the __main__.py.  Any help here to see what I'm missing would be greatly appreciated.  

Versions:
- OrthoFinder 2.5.2
- RAxML 8.2.12
- RAxML-NG 0.9.0

Operating System: CentOS 7.8

Also I noticed in the config.json that it's looking for "raxmlHPC-AVX", so I soft linked raxmlHPC-HYBRID-AVX to that.  I've confirmed that this is in my PATH.  Please let me know if any other information is needed and thanks so much for any advice.

**Comment by davidemms** [comment]
Hi

Try creating a test alignment file and then from the terminal you're running OrthoFinder from try running the raxml command you've written in the config.json file and see if it runs correctly.

Best wishes
David

**Comment by ghbhc** [comment]
David,

Thank you very much for your help.  This helped me track down the problem.  To anyone reading this in the future that may make the same mistake I did, by doing what David suggested in the comment above, I found that RAxML gave this error:

"Error: you are running the parallel MPI program but only want to compute one tree
For the MPI version you must specify a number of trees greater than 1 with the -# or -N option"

So I was using the hybrid-AVX version of RAxML when I should've been using the pthreads-AVX version of RAxML (RAxML comes with a lot of different Makefiles to choose from).  Making that correction solved the issue and OrthoFinder is working for me now.
